.github/workflows/black.yml

```diff
@@ -24,4 +24,4 @@ jobs:
         python -m pip install black
     - name: Lint
       shell: bash
-      run: black optking --check -l 120
+      run: black optking -l 120
```

Comment:
I was blindly hoping this would work but the official solution from black is to run black check and then commit and push a fixup

```
     - name: Lint
        run: black optking --check -l 120
        if: failure()
        run: |
          black optking -l 120
          git config --global user.name 'autoblack'
          git config --global user.email 'cclauss@users.noreply.github.com'
          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/$GITHUB_REPOSITORY
          git checkout $GITHUB_HEAD_REF
          git commit -am "fixup: Format Python code with Black"
          git push
```

Reply:
So what you have pasted here is what I should add to black.yml ?

Reply:
Yes. just read through it again - I don't think there are any errors :wink: . Just replace the `-name Lint` section with this.